### ISSUE TITLE
GraphViz attributes containing colon cause issues

### ISSUE THREAD

**Issue by tdene** [tdene/synth_opt_adders#94]
See pydot/pydot#258

It is strange that this issue occurs on some builds, but not others. Colab installs are fine, as is my local install.<br>
Yet this issue does occur for others.

**Comment by tcal-x** [comment]
I hit this issue on my personal laptop, which is running Ubuntu 20.04.   I am using `venv`, and this is what I have installed:
```
$ pip3 list
Package       Version
------------- -------
graphviz      0.20   
networkx      2.8.4  
Pillow        9.2.0  
pip           20.0.2 
pkg-resources 0.0.0  
pptrees       1.0.10 
pydot         1.4.2  
pyparsing     3.0.9  
setuptools    44.0.0 
```

P.S. the CI tests for #95 just need your click to kick off...

**Comment by tdene** [comment]
My sincere apologies for letting this sit for so long.

Part of it is new job stress, which I have now overcome.<br>
Most of it is that I wanted to fix #99 in the same version number as this, but #99 proved difficult to debug.

I am now active once again!
